charters/DataProcessingService/DataProcessingPipelines.md

```diff
@@ -0,0 +1,70 @@
+# [HCA Data Processing Pipelines](mailto:pipelines-team@data.humancellatlas.org)
+
+## Description
+The HCA Data Processing Pipelines are cloud-based and portable analysis pipelines that transform the data in the Human Cell Atlas.
+
+## Objectives
+The data processing service aims to develop scientific analysis pipelines for all analyses required on all scientific data types in the Human Cell Atlas Data Coordination Platform (DCP). Pipelines for additional data types, analyses, and improvements to these pipelines will be added iteratively and in collaboration with the HCA Analysis Working Group (AWG) and the community. 
+
+## Definitions
+__Community pipeline:__ A pipeline that is not officially used as a standard in the HCA project, but runs in the DCP and is operated by the Data Processing Service to enable community members to work with a diversity of submitted data until they have a standardized pipeline. These pipelines are not as vetted as standardized pipelines.  
```

Comment:
I would prefer not to use that language. The community should give feedback, thoughts, and contribute to pipelines but their convergence does not make a standard HCA pipeline, AWG approval does.
This should be one pipeline per assay because it is to enable the data modality until an official pipeline is installed. (This is all to be learned but this is my first guess).